### PR TITLE
Pmm 12499 inventory basic cluster view

### PR DESCRIPTION
**Pull request by rasika-chivate**
For details, see 

https://jira.percona.com/browse/PMM-12378

### CODE REVIEW

File: docs/details/dashboards/dashboard-inventory.md [hunk 1]
```diff
@@ -56,6 +57,17 @@ Understanding the relationship between nodes and services is key to gaining insi
 
 - **Incident response**: When an issue or incident occurs, understanding the node-service relationship helps in troubleshooting. You can quickly identify which nodes and services are affected and focus your efforts on resolving the problem.
 
+#### Cluster view
+
+Starting with PMM 2.40.0, you can effortlessly group services into clusters based on the labels with the **Organize by Clusters** toggle.
```
Comment (atymchuk) [inline docs/details/dashboards/dashboard-inventory.md:66]:
```suggestion
Starting with PMM 2.40.0, you can choose to view a group of services as a single cluster  with the **Organize by Clusters** toggle. PMM uses the `cluster` label to display services under the same cluster.
```

**Review by atymchuk — COMMENTED** [review]
I noticed the image `docs/_images/PMM_Inventory_Service_services_view.png` is uploaded by not used anywhere. Let's remove unused images since they add quite a lot to GitOps.

**Comment by rasika-chivate** [comment]
> PMM_Inventory_Service_services_view.png

Done

**Review by atymchuk — APPROVED** [review]
🚢

**Pull request merged**